lib/ddtrace/propagation/distributed_headers/base.rb

```diff
@@ -0,0 +1,21 @@
+require 'ddtrace/configuration'
+require 'ddtrace/span'
+require 'ddtrace/ext/priority'
+
+module Datadog
+  module DistributedHeaders
+    # Base provides common methods for distributed header helper classes
+    module Base
```

Comment:
We should pay careful attention to the use of `Base`, as it often represents an anti-pattern or ambiguous intent. What's the responsibility of this module, and can it be given a more descriptive name that matches that purpose?

Reply:
I think i originally thought more would be in here, but I really just need this one helper function to be accessible, definitely doesn't need to be this specific pattern.